partiql-ast/src/main/java/org/partiql/ast/graph/GraphDirection.java

```diff
@@ -11,6 +11,8 @@
 
 /**
  * Represents graph edge direction.
+ * <p>
+ * Note: this is an experimental API and subject to change without prior notice.
```

Comment:
`<p>` requires a trailing `</p>`. You might be seeing some Javadoc warnings without it if I'm not mistaken.